hueniversitypy/theme_mcgill.py

```diff
@@ -16,7 +16,6 @@ def theme_mcgill():
 
```

Comment:
Attribution should be added from the article that we used for reference here in the docstring